### PR TITLE
[4.0] Typo in the tinymce ini

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Fix a typo in the tinymce ini, (too much spaces :D)

### Testing Instructions

make sure the exteneral plugins description is still showed.

### Expected result

it is

### Actual result

it is with to much spaces :D

### Documentation Changes Required

none

### CODE REVIEW

File: administrator/language/en-GB/plg_editors_tinymce.ini [hunk 1]
```diff
@@ -74,6 +74,6 @@ PLG_TINY_XML_DESCRIPTION="TinyMCE is a platform independent web based JavaScript
 ; External Plugins
 PLG_TINY_FIELD_LABEL_EXTERNAL="External Plugins"
 PLG_TINY_FIELD_EXTERNALPLUGINS_LABEL="External Plugin URLs"
-PLG_TINY_FIELD_EXTERNALPLUGINS_DESC = "Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or a complete absolute URL."
+PLG_TINY_FIELD_EXTERNALPLUGINS_DESC="Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or a complete absolute URL."
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_editors_tinymce.ini]:
```suggestion
PLG_TINY_FIELD_EXTERNALPLUGINS_DESC="Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or an absolute URL."
```

by definition an absolute url is complete. so you dont need that word

Comment (zero-24) [reply administrator/language/en-GB/plg_editors_tinymce.ini]: thanks @brianteeman

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cadaa0b5139b8603d378e3adb16ca3117f437cd7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27305">issues.joomla.org/tracker/joomla-cms/27305</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on cadaa0b5139b8603d378e3adb16ca3117f437cd7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27305">issues.joomla.org/tracker/joomla-cms/27305</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27305">issues.joomla.org/tracker/joomla-cms/27305</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**